staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -509,8 +511,9 @@ func isDestRelative(base, dest string) bool {
 }
 
 func getPrefix(file string) string {
-	// tar strips the leading '/' if it's there, so we will too
-	return strings.TrimLeft(file, "/")
+	// tar strips the leading '/' and '\\' if it's there, so we will too
+	prefix := strings.TrimLeft(file, "/")
```

Comment:
should be `strings.TrimLeft(file, "/\\")`, the current code wouldn't modify `\/foo`